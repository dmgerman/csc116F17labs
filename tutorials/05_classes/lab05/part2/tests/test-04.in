25:30
1:35