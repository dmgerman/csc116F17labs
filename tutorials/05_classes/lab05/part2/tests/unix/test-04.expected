Enter a time: 
You entered "25:30"
Runtime error: Invalid time
